### PR TITLE
HBASE-22516 - changing versions values on related pom files

### PR DESCRIPTION
**Pull request by wchevreuil**
Pull request for the actual alpha version.

### CODE REVIEW

**Comment by wchevreuil** [comment]
Got it! Just committed new changes with version value now 1.0.0-alpha1.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# HBASE Changelog
```
Comment (busbey) [inline CHANGELOG.md]: this file needs a ASF license header. if you run release doc maker with the `--license` option it should do it for you.

File: RELEASENOTES.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# HBASE  hbase-filesystem-1.0.0-alpha1 Release Notes
```
Comment (busbey) [inline RELEASENOTES.md]: this file needs a ASF license header. if you run release doc maker with the `--license` option it should do it for you.

**Comment by wchevreuil** [comment]
Committed new version for CHANGELOG.md and RELEASENOTES.md, with ASF license header added.

**Pull request merged**